editor/src/messages/portfolio/menu_bar/menu_bar_message_handler.rs

```diff
@@ -151,13 +151,21 @@ impl PropertyHolder for MenuBarMessageHandler {
 							..MenuBarEntry::default()
 						},
 					],
-					vec![MenuBarEntry {
-						label: "Delete Selected".into(),
-						icon: Some("Trash".into()),
-						shortcut: action_keys!(DocumentMessageDiscriminant::DeleteSelectedLayers),
-						action: MenuBarEntry::create_action(|_| DocumentMessage::DeleteSelectedLayers.into()),
-						..MenuBarEntry::default()
-					}],
+					vec![
+						MenuBarEntry {
+							label: "Delete Selected".into(),
+							icon: Some("Trash".into()),
+							shortcut: action_keys!(DocumentMessageDiscriminant::DeleteSelectedLayers),
+							action: MenuBarEntry::create_action(|_| DocumentMessage::DeleteSelectedLayers.into()),
+							..MenuBarEntry::default()
+						},
+						MenuBarEntry {
+							label: "Center View on Selected".into(),
+							shortcut: action_keys!(NavigationMessageDiscriminant::FitViewportToSelection),
+							action: MenuBarEntry::create_action(|_| NavigationMessage::FitViewportToSelection.into()),
+							..MenuBarEntry::default()
+						},
```

Comment:
Done. Is there a method to disable this menu option when there are no items selected?

Reply:
Also, wondering would this menu option make more sense under the "View" menu?

Reply:
Great question! Unfortunately we don't really have that infrastructure yet, which is why all our menu bar entries don't get grayed out at the appropriate times yet. We just have to make sure we don't do anything unexpected (crash, throw errors, etc.) when it's called at an inappropriate time. The way to properly deal with this is with some kind of event-based model that should automatically update the UI according to changes in context. Otherwise it would require lots of message dispatching from dozens of disparate locations all to update specific parts of the UI which is fragile and super easy to do incorrectly. So we really need that infrastructure to update the UI based on contextual events.

Reply:
> Also, wondering would this menu option make more sense under the "View" menu?

You're right, I wasn't thinking. Let's call it "Zoom to Selected" and put it in the first slot of the View menu. Thanks for asking about that!